core/src/main/java/io/confluent/connect/storage/StorageSinkConnectorConfig.java

```diff
@@ -353,6 +390,10 @@ public String getAvroCodec() {
     return getString(AVRO_CODEC_CONFIG);
   }
 
+  public String parquetCodec() {
+    return getString(PARQUET_CODEC_CONFIG);
+  }
```

Comment:
This method will throw an exception if the config key was not defined. How about either documenting that or checking whether it is defined first?